app/views/sessions/patient_specific_directions/new.html.erb

```diff
@@ -6,7 +6,7 @@
                                         ]) %>
 <% end %>
 
-<%= h1 "Are you sure you want to add #{@eligible_for_bulk_psd_count} new PSDs?" %>
+<%= h1 "Are you sure you want to add #{@eligible_for_bulk_psd_count} new #{"PSD".pluralize(@eligible_for_bulk_psd_count)}?" %>
```

Comment:
I can see there are few other places where we say `PSDs` (found by searching for that string), do we need to update it in those other places as well?